types/chromecast-caf-receiver/cast.framework.breaks.d.ts

```diff
@@ -52,24 +67,29 @@ export interface BreakManager {
     getPlayWatchedBreak(): boolean;
 
     /**
-     * Provide an interceptor to allow developer to insert more break clips or modify current break clip before a break is started.
+     * Provide an interceptor to allow developer to insert more break clips or
+     * modify current break clip before a break is started.
      * If interceptor is null it will reset the interceptor to default one.
      * By default VAST fetching and parsing logic in default interceptor.
-     * So if customized interceptor is set by developer;
-     * the VAST logic will be overridden and developers should implement their own VAST fetching and parsing logic in the provided interceptor.
+     * So if customized interceptor is set by developer, the VAST logic will be
+     * overridden and developers should implement their own VAST fetching and
+     * parsing logic in the provided interceptor.
      */
     setBreakClipLoadInterceptor(
-        interceptor: (
+        interceptor: ((
             breakClip: BreakClip,
             breakClipLoaderContext?: BreakClipLoadInterceptorContext
-        ) => void
+        ) => void) | null
     ): void;
 
     /**
      * Provide an interceptor for developer to specify what breaks they want to play after seek.
+     * @param seekInterceptor Interceptor or null if developer wants to reset it
+     *                        to default one. The default break seek interceptor will
```

Comment:
Why the extra space?

Reply:
Since jsdoc is inherited from JavaDoc, I used the same convention: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide

If you prefer we can adopt [Google Javascript Styleguide](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide) that suggests four spaces.